flight/mcu/lib/teensyShared/rocketFSM.cpp

```diff
@@ -1,3 +1,16 @@
+/**
+ * @file        rocketFSM.cpp
+ * @authors     
+ *
+ * @brief      
+ *
+ * The implementation of the finite state machine class that governs state transitions.
+ * 
+ * 
```

Comment:
Here's a longer description:

The rocketFSM class encapsulates the finite state machine that dictates which state the rocket is in throughout the mission. The class implements the logic necessary to reliably transition between states along with hysteresis to avoid premature state transitions.

This is a highly flight critical software module and should be tested thoroughly in simulation and on hardware targets.